frontend/src/utils/constants.js

```diff
@@ -269,3 +269,10 @@ export const BadgeType = {
 }
 
 export const sidebarExcludedPaths = [GeneralRoutes.HOME, GeneralRoutes.PASSWORD_RESET]
+
+export const ActivityFileName = {
+  [Activity.INFO]: 'info-task-configuration.json',
+  [Activity.TASK]: 'combat-task-configuration.json',
+  [Activity.SURVEY]: 'survey-task-configuration.json',
+  [Activity.EXPEDITION]: 'graph-task-configuration.json'
```

Comment:
Można by się zastanowić czy nie chcemy robić nazwy pobieranego pliku jako <nazwa aktywności>-<rodzaj aktywności>, bo teraz przy pobieraniu będziemy mieć wiele plików o tej samej nazwie

Reply:
No to problem prowadzącego + może sobie zmienić przy zapisie. Ciężko by było dać nazwę aktywności w nazwie pliku przy dodawaniu nowej aktywności bo wtedy by i tak byłby domyślny string z json editora co by nie zlikwidowało tego problemu.